woocommerce/woocommerce-ios
# SPE: Show "Add product variations" in more details action sheet

Closes: https://github.com/woocommerce/woocommerce-ios/issues/8842

## Description

This PR adds "Add product variations" option in "more details" bottom sheet on the product form.
It triggers the switch of product type to variable. The action is only displayed on simple products.

_Note:_ section divider from design wasn't added in this PR since it requires significant changes to a generic component.

## Testing

1. Build and run the app in debug/alpha mode.
2. Open product details for any non-variable product (or start product creation flow).
3. Tap "Add more details" on the bottom of the screen.
4. Confirm "Add product variations" option is displayed in the bottom.
5. Tap it, save the changes and reopen the product to confirm.

## Screenshots

before|after
--|--
![Simulator Screen Shot - 1](https://user-images.githubusercontent.com/3132438/220066535-834d92bb-a6cd-42c9-82f8-33568ee6a037.png)|![Simulator Screen Shot - 2](https://user-images.githubusercontent.com/3132438/220066543-8715d143-fd90-4ed6-9c1b-f3b10342ad52.png)

---
